crates/state/src/exec_env.rs

```diff
@@ -19,7 +19,7 @@ pub struct ExecEnvState {
     /// seen on the corresponding DA layer yet.
     ///
     /// This must always be sorted.
-    waiting_da_blobs: Vec<da_blob::BlobSpec>,
+    waiting_da_blobs: Vec<BlobSpec>,
```

Comment:
This should stay as `BlobSpec`, correct.  Maybe we have to bring it into this crate?

Reply:
I am not sure. As mentioned above, I've created another type `PayloadSpec` in the same module.